test/httpprovider.js

```diff
@@ -130,7 +132,7 @@ describe('web3-providers-http', function () {
                     method: 'POST',
                     credentials: 'omit',
                     headers: {
-                      'Content-Type': 'application/json'
+                    'Content-Type': 'application/json'
```

Comment:
I think deleting those whitespaces happened by mistake :) 
```suggestion
                      'Content-Type': 'application/json'
```